components/brave_wallet/common/zcash_utils_unittest.cc

```diff
@@ -167,4 +167,187 @@ TEST(ZCashUtilsUnitTest, ExtractTransparentPart) {
   }
 }
 
+TEST(ZCashUtilsUnitTest, GetOrchardRawBytes) {
+  // https://github.com/zcash/librustzcash/blob/zcash_address-0.3.1/components/zcash_address/src/kind/unified/address/test_vectors.rs#L832
+  {
+    auto orchard_raw_bytes = GetOrchardRawBytes(
+        "u1a84vn0qes8q3jhk7zxs2whd2p922far8kztqdapergs5ej8rarn53v5ddnd6t7e3l5ef"
+        "haefrhkptatnzq565nrpvf7kn2787gdvervmk08azp4qgehaew2zplkxkkyu36l3v7drg2"
+        "v",
+        false);
+    ASSERT_TRUE(orchard_raw_bytes);
+    constexpr std::array<uint8_t, kOrchardRawBytesSize> expected_raw_bytes = {
+        0x3b, 0x68, 0xc2, 0x9b, 0x4a, 0x13, 0x8b, 0x28, 0x9f, 0xea, 0x8b,
+        0x67, 0x95, 0xe6, 0x47, 0x59, 0xa7, 0xcd, 0x7c, 0x0a, 0xaf, 0x4b,
+        0xb9, 0x8e, 0xd3, 0x07, 0x99, 0x59, 0xb0, 0xbb, 0xa9, 0xb7, 0x61,
+        0x70, 0x4b, 0x6c, 0xfc, 0x14, 0x65, 0xad, 0x74, 0xbb, 0x05};
+    EXPECT_EQ(orchard_raw_bytes.value(), expected_raw_bytes);
+  }
+
+  // Wrong
+  {
+    auto orchard_raw_bytes = GetOrchardRawBytes("u11", false);
+    EXPECT_FALSE(orchard_raw_bytes.has_value());
+  }
+
+  // Unknown typecode
+  // https://github.com/zcash/librustzcash/blob/zcash_address-0.3.1/components/zcash_address/src/kind/unified/address/test_vectors.rs#L472
```

Comment:
```suggestion
  // https://github.com/zcash/librustzcash/blob/zcash_address-0.3.1/components/zcash_address/src/kind/unified/address/test_vectors.rs#L400
```

Reply:
rest of the test vectors LGTM - thanks for adding the comments to easily find them